### PR TITLE
Add scala-http-client library

### PR DESCRIPTION
**Pull request by hmil**

### CODE REVIEW

File: _data/library/scalalibs.yml [hunk 1]
```diff
@@ -64,6 +64,10 @@
       url: https://github.com/chandu0101/sri
       desc: 'Sri (Scala React interface) is a scalajs library to build truly native cross platform (mobile (ios and android) and web) applications.'
       dep: '"com.github.chandu0101.sri" %%% "web" % "0.4.0"'
+    - name: Scala Http Client
+      url: https://github.com/hmil/scala-http-client
+      desc: 'A cross-platform HTTP client implementation for the browser, nodejs and the jvm.'
```
Comment (sjrd) [inline _data/library/scalalibs.yml]: nodejs -> Node.js
jvm -> JVM

**Comment by sjrd** [comment]
I think the category "Client-server communication" is more appropriate for this library.

**Comment by sjrd** [comment]
Could you squash the two commits?

**Comment by hmil** [comment]
This should not be necessary anymore since github lets you do it from the web interface when you accept a PR now, right ?

**Comment by sjrd** [comment]
It still does not let me squash your commits *plus* create a merge commit. And I want the merge commit to associate commits with PRs.

**Comment by hmil** [comment]
Ok then

**Comment by hmil** [comment]
(github laggs right now, I pushed the changes but it may take a while to appear here)

**Comment by sjrd** [comment]
LGTM

**Pull request merged**